Sources/CreateOptions/ConfigOptions.swift

```diff
@@ -175,9 +175,6 @@ public struct ConfigOptions: ParsableConfiguration {
     /// Parses dates (e.g. `"2021-09-29"`) using [NaiveDate](https://github.com/CreateAPI/NaiveDate)
     @Option public var useNaiveDate: Bool = true
 
-    /// If enabled, uses `Int64` or `Int32` when specified.
-    @Option public var useFixWidthIntegers: Bool = false
```

Comment:
There's a helper in **Issue+Warnings.swift** that instructs the user how to replace this property. 

Please could you update the message to point to the new data types option 👍🏼